src/NetTopologySuite/Densify/Densifier.cs

```diff
@@ -159,7 +159,7 @@ protected override CoordinateSequence TransformCoordinates(
                 // prevent creation of invalid LineStrings
                 if (parent is LineString && newPts.Length == 1)
                 {
-                    newPts = new Coordinate[0];
+                    newPts = Array.Empty<Coordinate>();
```

Comment:
:page_facing_up: Same basic note about behavior change as in `MinimumBoundingCircle.cs`